media-gfx/iscan/iscan-3.62.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop flag-o-matic udev xdg-utils
+
+DESCRIPTION="EPSON Image Scan v3 for Linux"
+HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php https://gitlab.com/utsushi/utsushi"
+SRC_URI="https://support.epson.net/linux/src/scanner/imagescanv3/common/imagescan_${PV}.orig.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="graphicsmagick gui"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-libs/boost:=
+	media-gfx/sane-backends
+	media-libs/tiff
+	virtual/libusb:1
+	virtual/jpeg
```

Comment:
So if we care about sorting, `virtual/jpeg` should be before `virtual/libusb:1` :) .

Reply:
Sorted everything now.